src/locales/en.json

```diff
@@ -806,7 +806,7 @@
             "EditTtgMapTitle": "Map Tools / Filaments (Edit TTG Map)",
             "Encoder": "Encoder",
             "Extruder": "Extruder",
-            "Gate": "Gate",
+            "Flowrate": "FLOWRATE",
```

Comment:
pls make it uppersize in the frontend. you cannot be sure, that every translator also write it uppercase in the locale file.